src/components/Dashboard/TalkDashboard.vue

```diff
@@ -177,171 +177,187 @@ function scrollEventCards({ direction }: { direction: 'backward' | 'forward' })
-			<span class="secondary_text">
-				{{ t('spreed', 'Schedule a meeting with a colleague from your calendar') }}
-			</span>
-			<NcButton class="talk-dashboard__calendar-button"
-				variant="secondary"
-				:href="generateUrl('apps/calendar')"
-				target="_blank">
-				<template #icon>
-					<IconCalendarBlank :size="20" />
+		<div class="talk-dashboard__items">
+			<div class="event-section">
+				<template v-if="eventsInitialised && eventRooms.length > 0">
+					<h3 class="title">
+						{{ t('spreed', 'Upcoming meetings') }}
+					</h3>
+					<div class="talk-dashboard__event-cards-wrapper"
+						:class="{ 'forward-scrollable': forwardScrollable, 'backward-scrollable': backwardScrollable }">
+						<div ref="eventCardsWrapper"
+							class="talk-dashboard__event-cards"
+							@scroll.passive="updateScrollableFlags">
+							<EventCard v-for="eventRoom in eventRooms"
+								:key="eventRoom.eventLink"
+								:event-room="eventRoom"
+								class="talk-dashboard__event-card" />
+						</div>
+						<div class="talk-dashboard__event-cards__scroll-indicator">
+							<NcButton v-show="backwardScrollable"
+								class="button-slide backward"
+								variant="tertiary"
+								:title="t('spreed', 'Scroll backward')"
+								:aria-label="t('spreed', 'Scroll backward')"
+								@click="scrollEventCards({ direction: 'backward' })">
+								<template #icon>
+									<IconArrowLeft class="bidirectional-icon" />
+								</template>
+							</NcButton>
+							<NcButton v-show="forwardScrollable"
+								class="button-slide forward"
+								variant="tertiary"
+								:title="t('spreed', 'Scroll forward')"
+								:aria-label="t('spreed', 'Scroll forward')"
+								@click="scrollEventCards({ direction: 'forward' })">
+								<template #icon>
+									<IconArrowRight class="bidirectional-icon" />
+								</template>
+							</NcButton>
+						</div>
+					</div>
 				</template>
-				{{ t('spreed', 'Open calendar') }}
-			</NcButton>
-		</div>
-		<div class="talk-dashboard__chats">
-			<div class="talk-dashboard__unread-mentions">
-				<h3 class="title">
-					{{ t('spreed', 'Unread mentions') }}
-				</h3>
-				<ConversationsListVirtual v-if="filteredConversations.length > 0 || !conversationsInitialised"
-					class="talk-dashboard__conversations-list"
-					:conversations="filteredConversations"
-					:loading="!conversationsInitialised" />
-				<NcEmptyContent v-else
-					class="talk-dashboard__empty-content"
-					:name="t('spreed', 'All caught up!')"
-					:description="t('spreed', 'You have no unread mentions')">
-					<template #icon>
-						<IconAt :size="40" />
+				<LoadingPlaceholder v-else-if="!eventsInitialised"
+					type="event-cards" />
+				<Section v-else
+					class="event-section--empty"
+					wide
+					:title="t('spreed', 'Schedule meetings')"
+					:subtitle="t('spreed', 'You don\'t have any upcoming meetings')"
+					:description="t('spreed', 'Schedule a meeting with a colleague from your calendar')">
```

Comment:
We should avoid using 'with a colleague' here, as NC can be used for other purposes. `Schedule a meeting from your calendar` looks good enough to me.

It's also worth to mention in documentation, that:
- in Calendar, location should be set to Talk conversation;
- in Talk, there's a shortcut for it in every conversation.